### PR TITLE
Skip flaky ticker tests

### PR DESCRIPTION
**Pull request by janisz**
## Description

Skip failing tests for investigation. 

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4781** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4781" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4781?utm_source=stack-comment).

File: pkg/concurrency/retry_ticker_test.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func (f *afterFuncSpy) afterFunc(d time.Duration, fn func()) *time.Timer {
 }
 
 func TestRetryTickerCallsTickFunction(t *testing.T) {
+	t.Skip("ROX-14933: enable once it's fixed and stable on GHA")
```
Comment (msugakov) [inline pkg/concurrency/retry_ticker_test.go:52]:
```suggestion
	// TODO(ROX-14933): reenable this once fixed
	t.Skip("ROX-14933: enable once it's fixed and stable on GHA")
```

File: pkg/concurrency/retry_ticker_test.go [hunk 2]
```diff
@@ -109,6 +110,7 @@ func TestRetryTickerStop(t *testing.T) {
 }
 
 func TestRetryTickerStopsOnNonRecoverableErrors(t *testing.T) {
+	t.Skip("ROX-14933: enable once it's fixed and stable on GHA")
```
Comment (msugakov) [inline pkg/concurrency/retry_ticker_test.go:113]:
```suggestion
	// TODO(ROX-14933): reenable this once fixed
	t.Skip("ROX-14933: enable once it's fixed and stable on GHA")
```

**Review by msugakov — APPROVED** [review]
Now you need to remove `ROX-14933` from the PR title because CI won't let you merge it due to `// TODO(ROX-14933)` comments.

**Pull request merged**